src/trajectory_data_handling/sqlite_querying.cpp

```diff
@@ -88,15 +137,22 @@ namespace trajectory_data_handling {
         }
 
         switch(callback_type) {
-            case load_trajectory_information_into_datastructure:
-                rc = sqlite3_exec(m_db, query.c_str(), callback_datastructure, 0, &zErrMsg);
+            case load_original_trajectory_information_into_datastructure:
```

Comment:
Som følge af enum class. Se min kommentar i .hpp filen.

```suggestion
            case query_purpose::load_original_trajectory_information_into_datastructure:
```

Reply:
Jeg må ikke sætte en kommentar på linje 130, men jeres string kan være ref til const, således:

`void query_handler::run_sql(std::string const& query, query_purpose callback_type)`